### PR TITLE
new records

### PR DESCRIPTION
**Pull request by thea-m**
If you have more information on these persons, let me know

### CODE REVIEW

File: new/PRS14056DelormeE.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:id="n1"><forename>M. Émile</forename> <surname>Delorme</surname></persName>
+                    <nationality type="France"/>
+                    <floruit when="1911">M. Émile Delorme, a French official, acquired manuscripts on a trip to Ethiopia in 1911. 
+                        His collection, partly described by <persName ref="PRS4805Grebaut"/>, later became dispersed, with several of his
```
Comment (DenisNosnitsin1970) [inline new/PRS14056DelormeE.xml:51]: I do not know further details about him, but M. probably stands for "monsieur" only.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: new/PRS14056DelormeE.xml (hunk 1 above)
Comment (eu-genia) [reply new/PRS14056DelormeE.xml:51]: I also found that a certain Émile Delorme was vice-president of the [Conseil d'administration de la Silocoop in Tunis in 1929](https://gallica.bnf.fr/ark:/12148/bpt6k51422954/f4.image.r=delorme) (could well be the same one, but of course not 100%), just a curiosity

Comment (thea-m) [reply new/PRS14056DelormeE.xml:51]: Great, thank you. 
Could you give me a reference for Walda Maryam? I do not find him in the sources mentioned here.

Comment (eu-genia) [reply new/PRS14056DelormeE.xml:51]: I believe it was in a footnote in Grébaut

Comment (eu-genia) [reply new/PRS14056DelormeE.xml:51]: image

**Review by eu-genia — APPROVED** [review]

**Pull request merged**